README.md

```diff
@@ -1,4 +1,4 @@
-![Metorikku Logo](https://raw.githubusercontent.com/wiki/yotpoltd/metorikku/metorikku.png)
+![Metorikku Logo](https://raw.githubusercontent.com/wiki/yotpoltd/metorikku/metorikku.png) 
```

Comment:
Checkout :)

Reply:
WHYYYYYYY???

Reply:
It was the quickest way to trigger a build. Reverted.